pkg/plugins/httpresponsesender/http_response_sender.go

```diff
@@ -0,0 +1,65 @@
+package httpresponsesender
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"net/textproto"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// HTTPResponseSender implements backend.CallResourceResponseSender and
+// writes an HTTP response using an http.ResponseWriter given received
+// backend.CallResourceResponse's.
```

Comment:
No need for apostrophe 
```suggestion
// backend.CallResourceResponse(s).
```